cmd/cartesi-node/root.go

```diff
@@ -3,16 +3,21 @@
 
 package main
 
-import "github.com/spf13/cobra"
+import (
+	"time"
+
+	"github.com/spf13/cobra"
+)
 
 var rootCmd = &cobra.Command{
 	Use: "cartesi-node [reader|validator|full|no-backend]",
 	Run: func(cmd *cobra.Command, args []string) { cmd.Usage() },
 }
 
+const DefaultServiceTimeout = 1 * time.Minute
```

Comment:
This seems out of place.

Reply:
I wasn't really happy with it either. Accepting suggestions 😄

Reply:
I'll leave it in the file it is used, for now at least.